src/runtime/runtime.js

```diff
@@ -37,29 +37,27 @@
   var $preventExtensions = Object.preventExtensions;
   var $seal = Object.seal;
   var $isExtensible = Object.isExtensible;
+  var $apply = Function.prototype.call.bind(Function.prototype.apply)
 
-  function nonEnum(value) {
-    return {
-      configurable: true,
-      enumerable: false,
-      value: value,
-      writable: true
-    };
+  function $bind(operand, thisArg, args) {
+    // args may be an arguments-like object
+    var argArray = [thisArg];
+    for (var i = 0; i < args.length; ++i) {
+      argArray[i + 1] = args[i];
+    }
+    var func = $apply(Function.prototype.bind, operand, argArray);
+    return func;
   }
 
-  var method = nonEnum;
+  function $construct(func, argArray) {
```

Comment:
I have `construct` below as well. I coined this one `$construct` because of its relation to `Reflect.construct` and for similarity to `$apply` and `Prototype.function.apply`, etc.